bolt/lib/Passes/ProfileQualityStats.cpp

```diff
@@ -187,8 +188,10 @@ void printCallGraphFlowConservationStats(
   std::vector<double> CallGraphGaps;
 
   for (const BinaryFunction *Function : Functions) {
-    if (Function->size() <= 1 || !Function->isSimple())
+    if (Function->size() <= 1 || !Function->isSimple()) {
```

Comment:
I believe we should now allow non-simple functions participate in call graph flow conservation gap computation, as they might be important from the layout perspective, and we strive to make sure call graph profile is attached to/from them.